acstools/polarization_tools.py

```diff
@@ -1,85 +1,328 @@
-#!/usr/bin/env python
 """
-Toolkit for analyzing HST/ACS polarization data.
+Toolkit for analyzing HST/ACS WFC and HRC polarization data. For more information
+about ACS polarization data analysis, see section 5.3 of the ACS Data Handbook:
 
-CHANGE LOG
-----------
-2020-09-22: Creation. (T. Desjardins)
+https://hst-docs.stsci.edu/acsdhb/chapter-5-acs-data-analysis/5-3-polarimetry
 """
 
-import numpy as np
 import os
+import numpy as np
 import yaml
 
+from astropy import units
 from astropy.table import Table
+from astropy.utils.data import get_pkg_data_filename
+
```

Comment:
This is for star import support. Also used for API doc generation. Please fill in the rest and remove `...`.
```suggestion

__all__ = ['calc_stokes', 'calc_fraction', ...]

```

Reply:
You need to fix this one to get Sphinx to generate API properly. That would fix linkcheck. The `__all__` need to include only the functions and classes you added in `polarization_tool.py`.